domain-services/ifix-master-data-service/src/main/java/org/egov/validator/ChartOfAccountValidator.java

```diff
@@ -137,7 +142,9 @@ public void validateCreatePost(COARequest coaRequest) {
     }
 
     public void validateCoaCode(COASearchCriteria searchCriteria) {
-        List<ChartOfAccount> chartOfAccounts = coaRepository.search(searchCriteria);
+        List<ChartOfAccount> chartOfAccounts = new ArrayList<>();
+        		//coaRepository.search(searchCriteria);
```

Comment:
What was the need to comment this? 
This will start violating the unique coa code constraint.